### PR TITLE
chore: add demo fixtures for a11y testing

### PR DESCRIPTION
**Pull request by dkasper-was-taken**
### Additional details
Created test and fixtures w/ semantic, styled, and role implementations:
- Button
- Link
-  Text Input
- CheckBox
- Range Slider

These serve as examples to test against intelligent a11y pages

### Steps to test
- `yarn workspace @packages/driver cypress:open`
- select e2e
- select a spec from the a11y folder

### PR Tasks

- [x] Have tests been added/updated?
- [n/a] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [n/a] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?

### CODE REVIEW

File: packages/driver/cypress/fixtures/a11y/slider.html [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+          newValue = 10
+          break
+        default:
+          break
+      }
+
+      newValue = Math.max(Math.min(newValue, 10), 0)
+
+      if (newValue !== value) {
+        console.log({ newValue, value })
```
Comment (lmiller1990) [inline packages/driver/cypress/fixtures/a11y/slider.html]: Maybe not needed?

File: packages/driver/cypress/fixtures/a11y/slider.html [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+        class="styled-label"
+      >
+        Role Points
+      </label>
+      <div
+        role="slider"
+        id="test-a11y-role-slider-container"
+        aria-orientation="horizontal"
+        tabindex="0"
+        aria-valuemin="0"
```
Comment (lmiller1990) [inline packages/driver/cypress/fixtures/a11y/slider.html:82]: Interesting, I had no idea you could do an accessible slider using role - I've only used `range`, which I guess isn't acesssible?

**Review by lmiller1990 — APPROVED** [review]

**Review by jordanpowell88 — APPROVED** [review]

File: packages/driver/cypress/fixtures/a11y/slider.html (hunk 2 above)
Comment (dkasper-was-taken) [reply packages/driver/cypress/fixtures/a11y/slider.html:82]: I'm not sure if it _isn't_ accessible, but `range` is the superclass for the type, so the guidance is to use a more specific subclass.
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/range_role

File: packages/driver/cypress/fixtures/a11y/slider.html (hunk 1 above)
Comment (dkasper-was-taken) [reply packages/driver/cypress/fixtures/a11y/slider.html]: Good catch, removed!

**Pull request merged**